src/components/Testimonial.astro

```diff
@@ -0,0 +1,26 @@
+---
+    export interface Props {
+        mainImageAlt?: string;
+        quote: string;
+        authorInformation: string;
+        authorDescription: string;
+        page:string;
+    }
+
+    const { mainImageAlt = " ", quote, authorInformation, authorDescription, page } = Astro.props;
+---
+
+<section class={`testimonial testimonial--${page}`}>
+    <picture class="testimonial__shape-container">
+        <source media="(min-width: 1024px)" srcset={`/assets/images/${page}/testimonial/testimonial-shape-desktop.svg`}>
+        <img class="testimonial__shape" src={`/assets/images/${page}/testimonial/testimonial-shape-mobile.svg`} alt=" ">
+    </picture>
+    <figure class="testimonial__image-container">
+        <img src={`/assets/images/${page}/testimonial/testimonial.png`} alt={mainImageAlt} class="testimonial__image">
+    </figure>
+    <article class="testimonial__content">
+        <blockquote class="testimonial__quote"><p>{quote}</p></blockquote>
```

Comment:
We may need to add a Testimonial with more than 1 paragraph of text. So I recomend this to be a slot/children and if we need multiple paragraphs, we can just send them.

```suggestion
        <Testimonial
    authorInformation="Lynn Kohlmeier, MD, Adult Endocrinologist"
    authorDescription="Alexion-sponsored speaker"
    page = "home"
  >
        <p>Something</p>
        <p>Lorem ipsum.....</p>
  </Testimonial>

```